database/src/migrations/release.0.2/restoration.sql

```diff
@@ -1370,8 +1458,243 @@ COMMENT ON COLUMN system_user_role.revision_count IS 'Revision count used for co
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.treatment_unit_id IS 'System generated surrogate primary key identifier.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.treatment_unit_spatial_component_type_id IS 'System generated surrogate primary key identifier.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.name IS 'The name of the record.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.description IS 'The description of the record.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.geometry IS 'The containing geometry of the record.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.geography IS 'The containing geography of the record.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.geojson IS 'A JSON representation of the geometry that provides necessary details for shape manipulation in client side tools.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.create_date IS 'The datetime the record was created.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.create_user IS 'The id of the user who created the record as identified in the system user table.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.update_date IS 'The datetime the record was updated.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.update_user IS 'The id of the user who updated the record as identified in the system user table.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component.revision_count IS 'Revision count used for concurrency control.'
+;
+COMMENT ON TABLE treatment_unit_spatial_component IS 'Treatment unit spatial component persists the various spatial components that a treatment unit may include.'
+;
+
+-- 
+-- TABLE: treatment_unit_spatial_component_type 
 --
--- TABLE: user_identity_source
+
+CREATE TABLE treatment_unit_spatial_component_type(
+    treatment_unit_spatial_component_type_id    integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    name                                        varchar(300)      NOT NULL,
+    description                                 varchar(250),
+    record_effective_date                       date              NOT NULL,
+    record_end_date                             date,
+    create_date                                 timestamptz(6)    DEFAULT now() NOT NULL,
+    create_user                                 integer           NOT NULL,
+    update_date                                 timestamptz(6),
+    update_user                                 integer,
+    revision_count                              integer           DEFAULT 0 NOT NULL,
+    CONSTRAINT treatment_unit_spatial_component_type_pk PRIMARY KEY (treatment_unit_spatial_component_type_id)
+)
+;
+
+
+
+COMMENT ON COLUMN treatment_unit_spatial_component_type.treatment_unit_spatial_component_type_id IS 'System generated surrogate primary key identifier.'
+;
+COMMENT ON COLUMN treatment_unit_spatial_component_type.name IS 'Name of the First Nation.'
```

Comment:
first nation

Reply:
whats with the first nation?! will adjust...